library/core/src/panic/panic_info.rs

```diff
@@ -126,3 +150,41 @@ impl fmt::Display for PanicInfo<'_> {
         Ok(())
     }
 }
+
+impl<'a> PanicMessage<'a> {
```

Comment:
```suggestion
#[unstable(feature = "panic_info_message", issue = "66745")]
impl<'a> PanicMessage<'a> {
```

Reply:
That attribute is only necessary on the methods in the impl block, not on the impl block itself, right?

Reply:
make sense, i thought it works like cfg attribute